include/internal/common.h

```diff
@@ -112,4 +116,15 @@ static ossl_inline int ossl_is_absolute_path(const char *path)
     return path[0] == '/';
 }
 
+static ossl_inline int ossl_is_uri(const char *s)
+{
+    const char *x;
+    for (x=s; ossl_isalnum(*x); ++x);
+#ifdef _WIN32
+    if (x-s <= 1)
+        return 0;
+#endif
+    return x > s && HAS_PREFIX(x, "://");
```

Comment:
This is certainly not right as URIs do not have to have the `//`

Reply:
That's true. Basically my concern here is supporting compatibility with anyone using multiple relative paths in `SSL_CERT_DIR`.

If you agree it's rare enough not to be worth worrying about, I'll change it.